### PR TITLE
Feature page: Add documentation for 4.1 DC.

### PR DESCRIPTION
**Pull request by maorlipchuk**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mureinik @mykaul the reviewer if relevant)

### CODE REVIEW

**Comment by maorlipchuk** [comment]
This is the first draft for the 4.1 DC.

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+wiki_category: Feature|DataCenterV4_1
+wiki_title: Features/DataCenterV4_1
+wiki_last_updated: 2016-09-20
+feature_name: Data Center V4.1
+feature_modules: engine/vdsm
+---
+
+## Summary
+#
+# Data Center 4.1 which was introduced at oVirt 4.1 should support the following features:
```
Comment (mykaul) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: was -> is being

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#
+# * Storage Domain's version upgrade - Upgrading Data storage domains' version from V3 to V4
+# This should make the storage domains support qcow version 1.1 as well as the old qcow version 0.10.
```
Comment (mykaul) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: qcow -> qcow2.
Note that also qcow2 v1.1 is known as qcow2v3 (worth mentioning it).

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# * libgfapi
```
Comment (mykaul) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: Gluster libgfapi (perhaps add a link to http://staged-gluster-docs.readthedocs.io/en/release3.7.0beta1/Features/libgfapi/ ) ?

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# * copy via HSM and discard
```
Comment (mykaul) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: Two different features?

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+# * io=native
```
Comment (mykaul) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: Rephrase to 'io=native' in both block and file based storage, as it provides better performance overall'.

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+* Maor Lipchuk
+* Email <mlipchuk@redhat.com>
+
+## Current status
+
+* Development
+
+## General Functionality
+
+* Data Center 4.1 which was introduced at oVirt 4.1 should support the following features:
```
Comment (mykaul) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: was -> will be 
at -> with

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md (hunk 1 above)
Comment (maorlipchuk) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: done

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
+---
+title: DataCenterV4_1
+category: feature
+authors: mlipchuk
+wiki_category: Feature|DataCenterV4_1
+wiki_title: Features/DataCenterV4_1
+wiki_last_updated: 2016-09-20
```
Comment (rollandf) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: You should not have any "wiki" item here. These are for pages imported from old wiki.
I believe this is the reason that the Ci failed.

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+feature_name: Data Center V4.1
+feature_modules: engine/vdsm
+---
```
Comment (rollandf) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: Missing "feature_status" tag.
for example:
feature_status: Released in oVirt 4.1
feature_status: In Progress

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md (hunk 1 above)
Comment (maorlipchuk) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: done

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md (hunk 2 above)
Comment (maorlipchuk) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: rephrased it as follow:
This should make the storage domains support qcow2 version 3 which is compatible with 
qemu 1.1 as well as the old qemu version 0.10.

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md (hunk 3 above)
Comment (maorlipchuk) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: done

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md (hunk 4 above)
Comment (maorlipchuk) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: I'm not sure...
Should I separate them to copy and discard

Comment (maorlipchuk) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: Rephrased it for now to "copy/discard via HSM"

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md (hunk 5 above)
Comment (maorlipchuk) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: done

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md (hunk 6 above)
Comment (maorlipchuk) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: done

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md (hunk 7 above)
Comment (maorlipchuk) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: removed

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md (hunk 8 above)
Comment (maorlipchuk) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: done

File: source/develop/release-management/features/storage/DataCenterV4_1.html.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+#
+# Data Center 4.1 which is being introduced at oVirt 4.1 should support the following features:
+#
+# * Storage Domain's version upgrade - Upgrading Data storage domains' version from V3 to V4.
+# This should make the storage domains support qcow2 version 3 which is compatible with
+# qemu 1.1 as well as the old qemu version 0.10.
+# * Gluster libgfapi - http://staged-gluster-docs.readthedocs.io/en/release3.7.0beta1/Features/libgfapi/
+# * copy/discard via HSM
+# * 'io=native' in both block and file based storage, as it provides better performance overall.
+#
```
Comment (rollandf) [inline source/develop/release-management/features/storage/DataCenterV4_1.html.md]: Why all this block starts with # ? 
Take a look at the preview, it does not display well.

**Comment by rollandf** [comment]
:+1:

**Comment by mureinik** [comment]
Can you squash these please? I'm fine with incremental improvements, but I don't want to introduce something wrong and then its fix.

**Comment by maorlipchuk** [comment]
On Sun, Oct 2, 2016 at 12:11 AM, Allon Murienik <notifications@github.com>
wrote:

> Can you squash these please? I'm fine with incremental improvements, but I
> don't want to introduce something wrong and then its fix.
>

Done, see https://github.com/oVirt/ovirt-site/pull/477

**Pull request merged**